job.go

```diff
@@ -25,6 +25,7 @@ type Job struct {
 	nextRun           time.Time      // datetime of next run
 	scheduledWeekday  []time.Weekday // Specific days of the week to start on
 	dayOfTheMonth     int            // Specific day of the month to run the job
+	dayOfTheMonths    []int          // Specific days of the month to run the job
```

Comment:
this should be `daysOfTheMonth`

```suggestion
	daysOfTheMonth    []int          // Specific days of the month to run the job
```

Could we remove the `dayOfTheMonth int` and instead add the single value to the slice? Then the logic could check whether Len is 0, 1, or >1.